.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: podman-next ${{ github.sha }}
+  IMAGE_REGISTRY: quay.io/podman
+  REGISTRY_USER: ${{ secrets.QUAY_PODMAN_USERNAME }}
+  REGISTRY_PASSWORD: ${{ secrets.QUAY.PODMAN_PASSWORD }}
+  COPR_OWNER: rhcontainerbot
+  COPR_PROJECT: podman-next
+
+jobs:
+  wait-for-task:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Set up wait-for-copr
+      run: |
+        pip3 install git+https://github.com/packit/wait-for-copr.git@main
+
+    - name: Wait for successful podman-next build with the latest commit
+      run: |
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman `git rev-parse --short ${{ github.sha }}`
```

Comment:
It only accepts a short SHA?  That's somewhat surprising.

Minor: I believe it's the preferred style to use `$(...)` for command-substitution.  It's easier for old-eyes to see as opposed to back-ticks.

Reply:
wait-for-copr waits for an rpm build with some string in the rpm name-version-release. A podman-next rpm version-release looks like: `4.6.2-1.20230829112801465782.main.1594.g94969a063`, the last part containing the short sha.

Reply:
Ahhh okay.  That would be a detail pretty unclear to any other maintainer.  The next time you touch this file, would you mind adding a little comment to note that?